skyportal/utils/gcn.py

```diff
@@ -266,46 +280,61 @@ def get_skymap_url(root, notice_type, timeout=10):
 
 
 def is_retraction(root):
-    retraction = root.find("./What/Param[@name='Retraction']")
-    if retraction is not None:
-        retraction = int(retraction.attrib['value'])
-        if retraction == 1:
-            return True
+    if isinstance(root, dict):
+        retraction = root.get('retraction')
+    else:
+        retraction = root.find("./What/Param[@name='Retraction']")
+        if retraction is not None:
+            retraction = int(retraction.attrib['value'])
+            if retraction == 1:
```

Comment:
Idk if it's git but it looks like in the JSON case you don't return True based on anything (the if statement looking at the retraction value is in the else that's above). Is that on purpose?

Reply:
Fixed, I think.